framework/include/utils/InputParameters.h

```diff
@@ -769,6 +770,8 @@ InputParameters::rangeCheck(const std::string & full_name, const std::string & s
     return;
   }
 
+  fp.setEpsilon(0);
```

Comment:
I think you really need a comment here. It's very unintuitive why this line of code is here. Anyone else looking at this might think it's unecessary.